installation-bundle/src/Config/ParameterDumper.php

```diff
@@ -34,17 +34,24 @@ class ParameterDumper
 
     public function __construct(string $rootDir, Filesystem $filesystem = null)
     {
-        $this->rootDir = $rootDir;
         $this->filesystem = $filesystem ?: new Filesystem();
         $parameters = [];
 
-        foreach (['app/config/parameters.yml.dist', 'app/config/parameters.yml'] as $file) {
+        if (file_exists($rootDir.'/config/parameters.yml') || !file_exists($rootDir.'/app/config/parameters.yml')) {
+            $this->configFile = $rootDir.'/config/parameters.yml';
+            $files = ['config/parameters.yml.dist', 'config/parameters.yml'];
```

Comment:
This is wrong, as there can never be a `parameters.yml.dist` file in the `/config` folder. The `/config` folder exists since Symfony flex and there is no `parameters.yml.dist` file in Symfony flex.

Reply:
Symfony flex is not relevant here at all. We either support `parameters.yml.dist` or we don't. We use that file to setup an installation from deployment, where it contains the default values. Without them, I can't access the install tool to setup the database connection.

Reply:
But your `parameters.yml` should define everything that is in the `parameters.yml.dist` file, shouldn't it?

Reply:
I never ever deploy a parameters.yml, so there will be no such file on the initial installation. That's exactly what the .dist file is for…

Reply:
Sorry, but that is certainly not a supported use case. The `parameters.yml.dist` file is required for the old Incenteev Parameter Handler (see https://symfony.com/doc/3.3/configuration.html#the-special-parameters-yml-file) and we have stopped using that one years ago. You will have to find a new solution, embracing the Symfony 4 best practices.

Reply:
Why do we support the file at all then?

Reply:
Backwards compatibility of course. That is also why we still have to load the file if it exists in the `app/config` folder.

Reply:
Actually we can ignore the `parameters.yml.dist` file completely, as we no longer support Symfony 3 in Contao 4.8.

Reply:
The `parameters.yml.dist` file has always been a **template**. You were supposed to add all parameters one can or must configure in the `parameters.yml` there and properly comment on them. I still use it to make sure that when somebody checks out the repository, they know immediately what to configure by copying `parameters.yml.dist` to `parameters.yml` and adjusting the config to their needs. It has, however, never been meant to operate as a fallback even though that concept is well known from other projects (`phpunit.xml.dist`, `php_cs.php.dist`, `.env`, etc.). So that file must not be loaded in the container. It can, however, serve as some kind of helper for something like the Incenteev Parameter Handler etc. Just not for the container.

Reply:
see 7ddbd24872d51b148ccd3a6fb054f49a71294e6b